substrait.gemspec

```diff
@@ -0,0 +1,36 @@
+# -*- ruby -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative "lib/substrait/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "substrait"
+  spec.version = Substrait::VERSION
+  spec.homepage = "https://substrait.io/"
+  spec.authors = ["Sutou Kouhei"]
+  spec.email = ["kou@clear-code.com"]
```

Comment:
Is there any preferred author?
For example, Apache Arrow uses "Apache Arrow Developers" and "dev@arrow.apache.org": https://github.com/apache/arrow/blob/master/ruby/red-arrow/red-arrow.gemspec#L32-L33

Reply:
Up to you. You can put `Substrait developers` or leave it as is.

Reply:
OK. I leave this as is because I want to maintain this.